projects/batfish/src/main/antlr4/org/batfish/grammar/flatjuniper/FlatJuniperLexer.g4

```diff
@@ -2693,6 +2689,14 @@ SPF_OPTIONS: 'spf-options';
 
 SPOOFING: 'spoofing';
 
+SQLNET_V2: 'sqlnet-v2';
```

Comment:
I gave up on having the bot alphabetize, probably it's failing for a variety of reasons but part of it is that the file is not already alphabetized well. You also see a lot of whitespace/newline changes b/c the file is already inconsistent. I decided better to make small fixes near places I touched (and that the bot initially put out of order).